MenuExample/Controller/MenuViewController.swift

```diff
@@ -19,6 +19,7 @@ class MenuViewController: UITableViewController {
     override func viewWillAppear(animated: Bool) {
         super.viewWillAppear(animated)
 
+        self.view.backgroundColor = UIColor.clearColor()
```

Comment:
@artemSydorenko this line of code is not related to the code upgrade to the swift 2.0. Do we really need this line ?